### DISCUSSION TITLE
Support for cross-platforms

### DISCUSSION THREAD

**Discussion by ArvindParekh** [YetAnotherPomodoroApp/YAPA-2#105] [General]
Is it, in any way, possible to extend the application for Linux users?

**Comment by floatas** [comment]
For reference, we have two issues about cross-platform: #76 and #20
It might be possible, would need to investigate a few frameworks and see how much work is needed: https://avaloniaui.net/ https://platform.uno/

**Reply by ArvindParekh** [reply]
Right okay, I get it.
I'm sorry I don't have much idea about how to implement cross-platform applications by moving some functionalities but thanks for the reply.
I'd just like to say that YAPA is a great productivity app and it'd be super useful to get it for other platforms too. Maybe somebody's willing to work on this, that is, if time permits. Thanks!

**Reply by floatas** [reply]
All good, I wrote it more for myself as a reference. I would love to make yapa cross-platform and maybe even as a web application.
